src/lib.rs

```diff
@@ -24,16 +13,15 @@ pub mod state;
 
 #[cfg(feature = "log-cost")]
 mod cu;
+#[cfg(not(feature = "no-entrypoint"))]
+mod entrypoint;
```

Comment:
Even the new module is named `entrypoint`, it has more stuffs than merely the entrypoints. 

I think `fast_process_instruction` and `slow_process_instruction` should not be behind `"no-entrypoint"` flag.


So it's better to add 3 lines in `lib.rs`:

```rust
+ #[cfg(not(feature = "no-entrypoint"))]
  entrypoint::custom_heap_default!();

+ #[cfg(not(feature = "no-entrypoint"))]
  entrypoint::custom_panic_default!();

+ #[cfg(not(feature = "no-entrypoint"))]
  pub unsafe extern "C" fn entrypoint(input: *mut u8)
```

That'd be minimal change to fix it.